IntelliTect.TestTools.TestFramework/ExampleTests/ExampleTests/ExampleTests.csproj

```diff
@@ -1,17 +1,30 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
-
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <LangVersion>9.0</LangVersion>
+    <Nullable>enable</Nullable>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <WarningLevel>4</WarningLevel>
+    <NoWarn>
+      CA1303;CS1822;CS1822;<!-- Next ones are temporary. -->CS8620;CS8625;CS8600;CS8602;CS8618;CS8603;CS8604;CA1822;CA1062
```

Comment:
Self note: Oops. Revisit these and remove from NoWarn if appropriate. Definitely remove comment when resolved.

Reply:
Will leave as-is and remove comment as part of a targeted cleanup effort.